packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -90,6 +90,12 @@ contract L2OutputOracle is Initializable, Semver {
         address _proposer,
         address _challenger
     ) Semver(1, 0, 0) {
```

Comment:
Do you mind adding the same check here?
https://github.com/ethereum-optimism/optimism/blob/3969dc495df37e2ef246aeb02e629f3238e0e7e7/packages/contracts-bedrock/deploy/012-L2OutputOracleImpl.ts#L12

Then the deployment will fail in the typescript instead of deploying a reverting contract

Reply:
Sure! Added the check to the deployment script as well. Still think we should retain the check in the contracts in case anyone who forks the stack chooses to go with a different route for deployment, doesn't add much to the deployment cost and prevents the proposer from being bricked.